server/routes/service.test.js

```diff
@@ -13,35 +14,35 @@ describe('Serivice layer unit test', () => {
   });
 
   // Test for function checkAuthStatus
-  it('Check an user authentication status to return string 200 for success', () => {
+  it('Should return string 200 for success', () => {
     // Arrange
-    sinon.stub(jwt, 'verify').returns('200');
+    sinon.stub(jwt, 'verify').withArgs('correct_info').returns('200');
     // Action
     const result = service.checkAuthStatus('correct_info');
     // Assert
     expect(result).to.equal('200');
   });
 
-  it('Check an user authentication status to return string 403 for failure', () => {
-    sinon.stub(jwt, 'verify').returns('403');
+  it('Should return string 403 for failure', () => {
+    sinon.stub(jwt, 'verify').withArgs('incorrect_info').returns('403');
 
     const result = service.checkAuthStatus('incorrect_info');
 
     expect(result).to.equal('403');
   });
 
   // Test for function setToken
-  it('Check an error as expected on generating an access token', () => {
-    sinon.stub(jwt, 'sign').returns(Error());
+  it('Should be an error for generating an access token', () => {
```

Comment:
`should return an error when jwt.sign fails` or `should return an error when jwt.sign returns an error`